src/VisualStudio/Core/Def/ColorSchemes/VisualStudio2019.xml

```diff
@@ -537,6 +552,12 @@
             <Color Name="xml literal - text">
                 <Foreground Type="CT_RAW" Source="FF3F3F3F" />
             </Color>
+            <Color Name="roslyn test code">
+                <Background Type="CT_RAW" Source="FF505050" />
```

Comment:
Should this be lighter

```suggestion
                <Background Type="CT_RAW" Source="FFE5E5E5" />
```

Reply:
I'll check. But I think I was following the exact colors that razor sets. I'll link you to that when I get home